timesketch/api/v1/resources.py

```diff
@@ -1064,6 +1080,148 @@ def post(self):
         return jsonify(self._get_info(aggregator_name))
 
 
+class AggregationGroupResource(ResourceMixin, Resource):
+    """Resource for aggregation group requests."""
+
+    @login_required
+    def get(self, sketch_id, group_id):
+        """Handles GET request to the resource.
+
+        Args:
+            sketch_id: Integer primary key for a sketch database model.
+            group_id: Integer primary key for an aggregation group database
+        """
+        sketch = Sketch.query.get_with_acl(sketch_id)
+        group = AggregationGroup.query.get(group_id)
+
+        if not group:
+            abort(
+                HTTP_STATUS_CODE_NOT_FOUND, 'No Group found with this ID.')
+
+        if not sketch:
+            abort(
+                HTTP_STATUS_CODE_NOT_FOUND, 'No sketch found with this ID.')
+
+        # Check that this group belongs to the sketch
+        if group.sketch_id != sketch.id:
+            msg = (
+                'The sketch ID ({0:d}) does not match with the aggregation '
+                'group sketch ID ({1:d})'.format(sketch.id, group.sketch_id))
+            abort(HTTP_STATUS_CODE_FORBIDDEN, msg)
+
+        if not sketch.has_permission(user=current_user, permission='read'):
+            abort(
+                HTTP_STATUS_CODE_FORBIDDEN,
+                'The user does not have read permission on the sketch.')
+
+        result_chart = None
+        how = group.how
```

Comment:
how? how what? :)

Reply:
Is this only for chart orientation? If yes, can we call this chart_orientation instead?

Reply:
how is `how should the charts be combined`... and yes, we can call it that instead... but `chart_orientation` is a pretty long parameter... how about just `orientation` instead of how?